src/Sulu/Bundle/WebsiteBundle/Twig/Seo/SeoTwigExtension.php

```diff
@@ -73,10 +83,18 @@ public function renderSeoTags(array $seoExtension, array $content, array $urls,
             $webspaceKey = $webspace->getKey();
         }
 
+        $request = $this->requestStack->getCurrentRequest();
```

Comment:
@webmozart WDYT about using the request like this in twig extensions? We need it to know if the links in this extension should be generated with `https://` or `http://`.